CONTRIBUTING.md

```diff
@@ -19,6 +19,14 @@ Or the linter:
 $ npm run lint
 ```
 
+If you want to run a single unit test, you can globally install the test runner
+and point it at the file you're interested in
+
+```sh
+$ npm i -g mocha
+$ mocha -t 5000 path/to/your/test.js
```

Comment:
Oh, I've just been doing `$(npm bin)/mocha path/to/test.js`.  Guess that's the same.

Nice addition to the docs!